services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core/_events.py

```diff
@@ -489,6 +487,7 @@ async def action(cls, app: FastAPI, scheduler_data: SchedulerData) -> None:
             project_id=scheduler_data.project_id,
             node_id=scheduler_data.node_uuid,
             simcore_user_agent=scheduler_data.request_simcore_user_agent,
+            swarm_stack_name=dynamic_sidecar_settings.SWARM_STACK_NAME,
```

Comment:
Are you sure this is required? Or do we ant to add this to the labels of the user services as well?

Reply:
I want to have the same everywhere, these are standard labels.